mesonbuild/interpreter/interpreter.py

```diff
@@ -107,7 +107,7 @@
 
     from typing_extensions import Literal
 
-    from . import kwargs
+    from . import kwargs as kwargs_
```

Comment:
Maybe something more descriptive kwanno or kwtypes?

Reply:
I thought about `T_kwargs` but we also have `TYPE_kwargs`...